Jenkinsfile

```diff
@@ -828,6 +828,7 @@ pipeline {
                     }
                     post {
                         always {
+                            junit testResults: 'bundles/junit-report.xml', allowEmptyResults: true
```

Comment:
@vikramhh hmm...i don't see the test results for RS5: https://ci.docker.com/public/job/moby/job/PR-39971/6/testReport/

does windows care about the direction of the slashes in filename paths?